src/hospital/Patient.java

```diff
@@ -0,0 +1,61 @@
+package hospital;
+
+import java.util.Date;
+
+/**
+ *
+ * @author maximdumont
+ */
+public class Patient {
+    public Patient(String firstName,String lastName,Date admittanceDate){
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.admissionDate = admittanceDate;
+    }    
+    
+    private Date admissionDate;
```

Comment:
Please move to top to keep with common conventions. :) We also need an ID field (int is fine for our usage) and a status field. We'd talked about using an enum or public static constants in the class. Or some other solution, not important so long as it works.